src/components/PivotTable/styles/PivotTable.style.js

```diff
@@ -9,21 +9,54 @@ import {
     FONT_SIZE_LARGE,
 } from '../../../modules/pivotTable/pivotTableConstants'
 
-export const table = css`
+export const table = css.global`
     div.pivot-table-container {
         font-family: 'Roboto', Arial, sans-serif;
         overflow: auto;
         color: ${colors.grey900};
     }
+
     table {
         border-spacing: 0;
-        border-collapse: separate;
         white-space: nowrap;
         box-sizing: border-box;
         text-align: center;
+        border: 1px solid #b2b2b2;
```

Comment:
ui color instead of hex value?

Reply:
There is no such color in `ui`, but I added a `const` instead.